src/client/hydrate.test.jsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import * as reactDom from 'react-dom';
-import { ClientApp } from './app/containers/App';
-import routes from './app/routes';
+import { ClientApp } from '../app/containers/App';
+import routes from '../app/routes';
+import hydrateClient from './hydrate';
 
 jest.mock('react-dom');
 
 jest.mock('react-router-dom');
 
-jest.mock('./app/containers/App');
+jest.mock('../app/containers/App');
 
-jest.mock('./app/routes', () => ({
+jest.mock('../app/routes', () => ({
   default: [],
 }));
 
+window.crypto = () => {};
```

Comment:
```suggestion
window.crypto = () => {}; // depended on by the cuts-the-mustard check
```

I'm presuming this is needed but worth double-checking if there's a way around it?